### PR TITLE
Adds a new, horrible race, the ugandan warrior.

### PR DESCRIPTION
**Pull request by Carbonhell**
:cl: 
add: Adds a new race, the ugandan warrior. I am sorry.
/:cl:

[why]: I don't know either, you tell me.
sprites with clothes and without
  https://imgur.com/a/vuaA0

### CODE REVIEW

**Comment by KayeArray** [comment]
pics

**Comment by tortellinitony** [comment]
it's knuckles the hedgehog but poorly sprited

**Comment by optimumtact** [comment]
do you know de wae?

**Comment by Carbonhell** [comment]
updated with pics of the sprites.

**Comment by Carbonhell** [comment]
current issues:masks do not work(and they may not be able to cover the entire head, need help)

**Comment by Carbonhell** [comment]
K so everything seems working, except masks like balaclava not hiding the horizontal edges of the face. This seems to be a bug not related to this PR,more of HIDEFACE not actually hiding face but just facial features, no clue why. Apart from that,the PR seems good to go, waiting for erik

**Comment by Wilchenx** [comment]
XDDD EPIK MAY MAY SPEEDMERGE NOW

**Comment by Wilchenx** [comment]
HOLY FUCK THIS NEEDS TO HAPPEN RIGHT NOWOWOW I'M CUMMING

**Comment by Wilchenx** [comment]
SPRITE SO GOOD MY BRAIN HURTS

File: hippiestation/code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -37,6 +37,8 @@
 				sound = 'hippiestation/sound/voice/caw.ogg'
 			if (is_species(user, /datum/species/tarajan))
 				sound = 'hippiestation/sound/voice/cat.ogg'
+			if (is_species(user, /datum/species/ugandan_warrior))
```
Comment (JohnGinnane) [inline hippiestation/code/modules/mob/living/emote.dm:40]: This reminds me to put screams into the species datum so we don't need to worry about this for future species

**Review by JohnGinnane — APPROVED** [review]
Looks good to me. Will it only ever be admin spawn only?

**Comment by JohnGinnane** [comment]
@HippieStation/head @Chnkr need head input

**Comment by Carbonhell** [comment]
so...any news?

**Comment by JohnGinnane** [comment]
carbon the meme is dead

**Comment by Carbonhell** [comment]
no it isn't
*cry

**Comment by ghost** [comment]
it's dead

**Comment by praisenarsie** [comment]
it's not dead just yet but it's starting to die and since i don't like the meme uhhhhhhhhh

**Comment by chesse20** [comment]
can we reopen this

**Pull request closed without merging**